src/lib/output/themes/default/templates/reflection.tsx

```diff
@@ -39,7 +39,7 @@ export function reflectionTemplate(context: DefaultThemeRenderContext, props: Pa
             {hasTypeParameters(props.model) && <> {context.typeParameters(props.model.typeParameters)} </>}
             {props.model instanceof DeclarationReflection && (
                 <>
-                    {context.hierarchy(props.model.typeHierarchy)}
+                    {context.hierarchy(props.model)}
```

Comment:
If this change is made it has to wait for 0.27 due to being a breaking change

Reply:
Would adding a second argument be considered a breaking change? That could work.

Reply:
If the second argument is required, yes, that's also a breaking change. You shouldn't need it as `context.page` is the `PageEvent` for the current page

Reply:
Fixed